x/liquidgov/keeper/hooks.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	epochstypes "github.com/Stride-Labs/stride/v5/x/epochs/types"
+	"github.com/Stride-Labs/stride/v5/x/liquidgov/types"
+)
+
+func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
+	if epochInfo.Identifier == epochstypes.STRIDE_EPOCH {
+		k.Logger(ctx).Info(fmt.Sprintf("%s RUNNING...", types.ModuleName)) // debugging
+
+		// on epoch update proposals
+		k.UpdateProposals(ctx)
+
+		// 	// check for outstanding proposals in vote cast window and cast votes
+		// 	k.CastVotes(ctx, epochInfo)
+
+		// // for every mature record
+		//
+		//	// either loop through records here or dequeue from UnlockingRecordQueue
+		//	k.CompleteUnlocking(ctx, denom, owner) // returns mature lockups to owners and deletes lockups
+	}
+}
+
+func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {}
+
+// Hooks wrapper struct for incentives keeper
+type Hooks struct {
+	k Keeper
+}
+
+var _ epochstypes.EpochHooks = Hooks{}
+
+func (k Keeper) Hooks() Hooks {
+	return Hooks{k}
+}
+
+// epochs hooks
+func (h Hooks) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
+	h.k.BeforeEpochStart(ctx, epochInfo)
+}
+
+func (h Hooks) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
+	h.k.AfterEpochEnd(ctx, epochInfo)
+}
+
+// // Update the epoch information in the stakeibc epoch tracker
```

Comment:
We can likely remove this I think, or use the EpochTracker from StakeIBC

Minor question, do you think it's better to use new epoch hooks in `x/liquidgov`, or call `x/liquidgov` functions from `x/stakeibc`? I don't have a strong view either way, curious what you think. I generally think it's slightly cleaner to have it in its own module (like you have here), but I think this is the first time we have epochly logic outside of `x/stakeibc`